src/web/components/MostViewed/MostViewedFooter/MostViewedFooterGrid.tsx

```diff
@@ -193,7 +193,9 @@ export const MostViewedFooterGrid = ({ data, sectionName, pillar }: Props) => {
 					aria-labelledby={`tabs-popular-${i}-tab`}
 					data-link-name={tab.heading}
 					data-testid={tab.heading}
-					data-link-context={`most-read/${sectionName}`}
+					data-link-context={
+						sectionName ? `most-read/${sectionName}` : ''
+					}
```

Comment:
Is there any impact to setting `data-link-context` as an empty string? Are we happy that the code reading this will work in this situation?